packages/thirdweb/src/react/web/wallets/in-app/InAppWalletFormUI.tsx

```diff
@@ -95,16 +102,8 @@ export const InAppWalletFormUI = (props: InAppWalletFormUIProps) => {
     type = "tel";
   }
 
-  const switchInputModeText =
-    inputMode === "email" ? locale.signInWithPhone : locale.signInWithEmail;
-
-  const switchInputMode = useCallback(() => {
-    setInputMode((prev) => (prev === "email" ? "phone" : "email"));
-  }, []);
-  const allowSwitchInputMode = isEmailEnabled && isPhoneEnabled;
-
   const socialLogins = authOptions.filter(
```

Comment:
Should we keep this and add `!== passkey` since we're more likely to add more social logins? (have heard Farcaster, X, and Discord mentioned)